builder/codegen/src/main/java/io/helidon/builder/codegen/CustomMethods.java

```diff
@@ -225,6 +232,46 @@ private static GeneratedMethod factoryMethod(Errors.Collector errors,
                                    codeGenerator);
     }
 
+    private static List<CustomConstant> findConstants(TypeInfo customMethodsType,
+                                                      Errors.Collector errors) {
+        return customMethodsType.elementInfo()
+                .stream()
+                .filter(ElementInfoPredicates::isField)
+                .filter(ElementInfoPredicates.hasAnnotation(PROTOTYPE_CONSTANT))
+                .map(it -> {
+                    if (!it.elementModifiers().contains(Modifier.STATIC)) {
+                        errors.fatal(it,
+                                     "A field annotated with @Prototype.Constant must be static, final, "
+                                             + "and at least package local. Not static.");
+                    }
+                    if (!it.elementModifiers().contains(Modifier.FINAL)) {
+                        errors.fatal(it,
+                                     "A field annotated with @Prototype.Constant must be static, final, "
+                                             + "and at least package local. Not final.");
```

Comment:
"Not final." should be removed as it is confusing.

Reply:
I will try to reword.